docs/src/developers_guide/contributing_getting_involved.rst

```diff
@@ -61,6 +61,6 @@ If you are new to using GitHub we recommend reading the
 
    ../generated/api/iris
    ../whatsnew/index
-   ../techpapers/index
+   ../../further_topics/techpapers/index
```

Comment:
Do we still want this? 

It means it'd still be accessible via Developers_Guide(>Sidebar)>References, and would just be a shortcut to it's new home in Further Topics.

I've left this in so it's easier for users to find, but it feels like it undermines the point of it perhaps?

Reply:
Personally I **_think_** I would have preferred that the old "tech papers" should just be merged into the"further topics" list.
But I note that this would mean there is nothing for that old whatsnew link to point **_at_**.  
Which is a problem I hadn't even thought of before this ... https://github.com/SciTools/iris/issues/5603

I don't think we can fix that problem here+now (!) -- it may want further discussion anyway.

So I think this is OK at least as an interim solution.
Or, if you fancy it, you could merge all topics but also create an overview page for the list, which could act as a target for this link.  @trexfeathers suggests that he'd like to have an overview page for the further topics anyway.

Reply:
No, I'd say we **_don't_** want this link any more.